index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+
+  <body>
+    <main id="main">
+      <h1 id="title">Percy Lavon Julian</h1>
+      <div id="img-div">
```

Comment:
The whole content after the header can be added to a section tag instead of div tag. This is semantic and would specify that they are grouped together if the reader is using reading ade tools, and of course for better code.